challenge-sizebay/src/utils/getCurrentMonth.jsx

```diff
@@ -0,0 +1,28 @@
+export const getCurrentMonth = (number) => {
+  switch (number) {
+    case 0:
+      return "January";
+    case 1:
+      return "February";
+    case 2:
+      return "March";
+    case 3:
+      return "April";
+    case 4:
+      return "May";
+    case 5:
+      return "June";
+    case 6:
+      return "July";
+    case 7:
+      return "August";
+    case 8:
+      return "September";
+    case 9:
+      return "October";
+    case 10:
+      return "November";
+    default:
+      return "December";
```

Comment:
ref: this can be improved in some ways.

if you're willing to keep this static, you could simply turn this into an array that you'll search based on the argument:

```js
const months = [
    'January',
    'February',
    'March',
    'April',
    'May',
    'June',
    'July',
    'August',
    'September',
    'October',
    'November',
    'December',
  ];

  return months[month];
```

There's also the potential to use the very-well-built Date API from JavaScript, like this:

```js
/**
 * Hey Luiz Fernando, check this out!
 * Get the current month, parsed from a number to a locale string.
 * @param month - The month to parse. 0 is January, 1 is February, etc. Make sure to subtract 1 from the month you want to get.
 * @returns
 */
function getCurrentMonth(month) {
  const date = new Date();
  date.setMonth(month);

  return date.toLocaleString('en-us', { month: 'long' });
}
```

Reply:
Done